lib/Backends/Habana/HabanaDeviceManager.h

```diff
@@ -138,6 +138,10 @@ class HabanaDeviceManager : public DeviceManager {
   uint64_t getMaximumMemory() const override;
   uint64_t getAvailableMemory() const override;
   bool isMemoryAvailable(uint64_t estimate) const override;
+
+  /// Returns the DeviceInfo for this device containing peak limits for
+  /// compute and bandwidths (used in partitioning).
+  DeviceInfo getDeviceInfo() const override;
```

Comment:
The DeviceConfigs is passed into HostManager for initialization. Basically, it contains all backend related information and then the DeviceManagers are created.  I don't think this function is necessary. The numbers used for roofline calculation should be added in to DeviceConfigs and passed to HostManager.